src/frontend/src/components/Post/Post.jsx

```diff
@@ -141,7 +144,13 @@ const Post = ({ postUrl }) => {
       <ListSubheader className={classes.header}>
         <AdminButtons />
         <Typography variant="h1" title={post.title} id={post.id} className={classes.title}>
-          {post.title}
+          <span onClick={expandTitleOrNot} style={{ cursor: 'pointer' }}>
```

Comment:
I think a cleaner way to accomplish all this is to isolate these different styles into two classes:

```js
expandHeader: {
  ...
},
collapseHeader: {
  ...
},
```

Now when you render this `<span>`, pick the classname to use:

```js
<span onClick={expandTitleOrNot} className={expandHeader ? 'expandHeader' : 'collapseHeader'}>...</span>
```